exonum/src/api/public/blockchain_explorer.rs

```diff
@@ -33,6 +33,13 @@ enum TransactionInfo {
     Committed(TxInfo),
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug)]
+struct ResponseInfo {
+    pub count: u64,
+    pub latest: Option<u64>,
+    pub skip_empty_blocks: Option<bool>,
```

Comment:
I'd rather try to use [`#[serde(default = ..)]`](https://serde.rs/attr-default.html) here; using a `null` value for `skip_empty_blocks` in JSON would be rather strange.